src/platform/packages/shared/kbn-unified-doc-viewer/src/components/field_name/field_name.tsx

```diff
@@ -45,16 +35,13 @@ export function FieldName({
   scripted = false,
   highlight = '',
 }: Props) {
-  const styles = useMemoCss(componentStyles);
-
   const typeName = getFieldTypeName(fieldType);
   const fieldMappingDisplayName = fieldMapping?.displayName ? fieldMapping.displayName : fieldName;
   const fieldDisplayName = displayNameOverride ?? fieldMappingDisplayName;
```

Comment:
If we are removing the abbreviation of the attributes field name, `displayNameOverride` needs to be removed from the props here
```suggestion
  const displayName =
    fieldMapping && fieldMapping.displayName ? fieldMapping.displayName : fieldName;
  const tooltip = displayName !== fieldName ? `${displayName} (${fieldName})` : fieldName;
```
Was also introduced in:
- src/platform/plugins/shared/unified_doc_viewer/public/components/doc_viewer_table/field_row.ts
- src/platform/plugins/shared/unified_doc_viewer/public/components/doc_viewer_table/table_cell.tsx